_data/blogs.yml

```diff
@@ -286,5 +286,4 @@
 - url: https://nishantgci.wordpress.com/2018/01/02/my-journey-in-open-source/
   title: My journey in the world of open-source programming
   img: pp.jpeg
-  author_name: Nishant S.
-
+  author_name: Nishant S.
```

Comment:
Please add a new line here.

Reply:
Where?
Should I put author_name in line 289 or should I delete line 289?

Reply:
Put a line after 289.

Reply:
But I was instructed to delete the line in the first commit.